package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "react-dev": "webpack -d --watch",
+    "build": "webpack",
```

Comment:
You might want to consider using production mode here, but this should be fine for now.

https://webpack.js.org/configuration/mode/

Reply:
Thanks! That is very good! Will do!